charts/umbrella/README.md

```diff
@@ -664,6 +664,12 @@ Build fuseki docker image by following the below steps:
 - Unzip the jena-fuseki-docker-4.7.0.zip.
 - Build the docker image by running the command - `docker build --build-arg JENA_VERSION=4.7.0 -t jena-fuseki-docker:4.7.0 --platform linux/amd64 .`
 
+## Precondition for IATP Mock
+
+in case of enabling `iatpmock` (e.g. by using [values-adopter-data-exchange.yaml](values-adopter-data-exchange.yaml)), the iatp-mock docker image must be built first:
+
+`docker build iatp-mock/ -t tractusx/iatp-mock:testing --platform linux/amd64`
```

Comment:
we should discuss if it's not worth to also publish the image, for e2e testing maybe not necessary but from a local sandbox (easy entrance point for new developers) maybe beneficial

Reply:
That's something I don't like either. The mock requires configuration in code which can change depending on the use-case. I did not manage to get this configurable via helm chart in a reasonable amount of time, so I decided to not publish the image. 
I'm open for suggestions here

Reply:
@tom-rm-meyer-ISST could you please have a look if it's possible to make the mock more configurable?